verifiers/internal/gcb/keys/keys.go

```diff
@@ -0,0 +1,58 @@
+package keys
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"embed"
+	"encoding/pem"
+	"fmt"
+	"io/fs"
+	"path"
+
+	serrors "github.com/slsa-framework/slsa-verifier/errors"
+)
+
+//go:embed materials/*
+var publicKeys embed.FS
+
+type PublicKey struct {
+	Value  []byte
+	Region string
```

Comment:
Any reason these need to be exported values?

Reply:
mistake from me, thanks for catching.